### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#745] [Galaxy Trivia] [answered]
**ITONLYPASTA**
**IDOTENNN**
**SIRRAOMEUP**
**MASTEKCED**

**Comment by peterbaccaro** [comment]
1. ATONALITY
2. INTONED
3. ARMOURIES
4. CASKETED

The odd one out is **ATONALITY**.

**Comment by ParanoidUser** [comment]
**Hint:** They could be two-word anagrams.

**Reply by peterbaccaro** [reply]
Any clues?

**Comment by peterbaccaro** [comment]
1. PLAYSTATION
2. NINTENDO
3. SUPER MARIO
4. DECK MATES

Based on the above the odd one out is MASTEKCED because it is not related to game consoles or video games.

**Comment by ParanoidUser** [comment]
**Another Hint** 😎🎮
![image](https://github.com/ParanoidUser/yolo/assets/5120290/c2b224e1-0f61-44ab-bece-16faf1b2a297)

**Reply by peterbaccaro** [reply, marked as answer]
1. ITONLYPASTA is **PLAYSTATION**
2. IDOTENNN is **NINTENDO**
3. SIRRAOMEUP is **SUPER MARIO**
4. MASTEKCED is **STEAM DECK**

Based on the above the odd one out is **SIRRAOMEUP (i.e. SUPER MARIO**) because it is a video game whereas the others are gaming devices.

**Reply by ParanoidUser** [reply]
Nailed it! 👏👏
